ChiModules/LinearBoltzmannSolver/Tools/kspmonitor_npt.cc

```diff
@@ -80,29 +61,30 @@ KSPConvergenceTestNPT(KSP ksp, PetscInt n, PetscReal rnorm,
 
 
   chi_log.Log(LOG_0) << iter_info.str() << std::endl;
-
-  if (context->groupset->iterative_method == NPT_GMRES)
+  const double SIXTY_SECOND_INTERVAL = 60000.0; //time in milliseconds
```

Comment:
how do we know the time was given in msec? 
also, isn't a sixty-second interval also a minute?  If so, why not call this factor: convert_to_minutes or something like that?